src/Items/ItemBigFlower.h

```diff
@@ -8,14 +8,14 @@
 
 
 
-class cItemBigFlowerHandler:
+class cItemBigFlowerHandler final:
 	public cItemHandler
 {
 	using Super = cItemHandler;
 
 public:
 
-	cItemBigFlowerHandler():
+	constexpr cItemBigFlowerHandler(int a_ItemType):
 		Super(E_BLOCK_BIG_FLOWER)
```

Comment:
should this be a_ItemType here, or the arg removed?

Reply:
You could say `using Super::Super` to pull in cItemHandler's constructor.

Reply:
I'm no C++ expert to know a better way, but moving the item ID to the construction doesn't strike me as being that big of an issue. The only problem here is the argument that does nothing, just pass it through.

Reply:
Yeah That's no problem. But I'd rather have a constructor without an argument.